pandas/tests/tseries/offsets/test_ticks.py

```diff
@@ -369,3 +369,8 @@ def test_compare_ticks_to_timedeltalike(cls):
         assert not off > other
         assert off <= other
         assert off >= other
+
+
+def test_offsets_hashable():
+    # GH: 37267
+    assert offsets.Day() is not None
```

Comment:
can you use `hash(offsets.Day())` and also use the offsets fixture to make sure this covers all

Reply:
Sorry, that was stupid by me. Done